packages/core/src/theme/tokens.json

```diff
@@ -1,80 +1,12 @@
 {
-  "aliases": {
```

Comment:
I had to separate this to a indipendent file, because otherwise theme importing `tokens.json` also processes the token props.

Reply:
That's fine, but the naming is confusing.
I see tokens as the actual value pointers/references, e.g. `token: 2px`
Then aliases reference that token, e.g. `alias: var(--token)`

Are you fine with swapping the name on these files, or do you have another suggestion?

Reply:
I'd like to keep the Theo naming...

Let's forget now the CSS tokens, this will also be used for iOS & Android, so e.g. even Android has ability to alias but iOS does not... It the outcome will be different on each platform.

Reply:
I see. Designers also refer to them as "design tokens", adding to the confusion.
Let's not use the word "alias" anymore then, in chat and verbally, we can call them "base/palette tokens" and "theme tokens".

Reply:
I would say these are Theo aliases.

Then we have base tokens, then theme tokens.

During generation of base tokens Theo aliases are used.

During generation of theme tokens also Theo aliases are used.

However from design perspective theme tokens alias to base tokens.

So I see this as separate terminology for design and engineering sides.